src/System.Device.Gpio/Interop/Unix/libgpiod/V2/GpiodException.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+// Disable these StyleCop rules for this file, as we are using native names here.
+#pragma warning disable SA1300 // Element should begin with upper-case letter
```

Comment:
As shared in a different comment, if we change the namespace to make it shorter we can also avoid needing to do this in every file. It is nice to have the namespace and library name match, but not a hard requirement.